ihmc-robotics-toolkit/src/test/java/us/ihmc/robotics/MatrixMissingToolsTest.java

```diff
@@ -48,6 +50,33 @@ else if (row == 3 && col == 5)
       }
    }
 
+   @Test
+   public void testNegate()
+   {
+      DMatrixRMaj matrixToNegate = new DMatrixRMaj(0, 0);
+      int iters = 100;
+      Random random = new Random(1738L);
+      for (int i = 0; i < iters; i++)
+      {
+         // reinitialize the matrix to random size and value
+         int initializeRows = RandomNumbers.nextInt(random, 100, 1000);
+         int initializeCols = RandomNumbers.nextInt(random, 100, 1000);
+         matrixToNegate.set(RandomMatrices_DDRM.rectangle(initializeRows, initializeCols, random));
```

Comment:
it's to initialize the backing data array to something of a different size, and with random data. It may be that it ends up being much larger. overwriting with mroe information only overwrites part of the data array.